RELEASING.md

```diff
@@ -1,7 +1,38 @@
+- [Overview](#overview)
+- [Branching](#branching)
+  - [Release Branching](#release-branching)
+  - [Feature Branches](#feature-branches)
+- [Release Labels](#release-labels)
+- [Releasing](#releasing)
+
+## Overview
+
+This document explains the release strategy for artifacts in this organization.
+
+## Branching
+
+### Release Branching
+
+Given the current major release of 1.0, projects in this organization maintain the following active branches.
+
+* **main**: The next _major_ release. This is the branch where all merges take place and code moves fast.
```

Comment:
Since main is tracking "next" major version which is 2.0.0 should version.txt in main be pointing to `2.0.0`?
Else you can modify this line to say main is tracking 1.x series. In that case you don't need `1.x` branch